### PR TITLE
Dev doha

### PR DESCRIPTION
**Pull request by dohmeid**

### CODE REVIEW

File: css/list.css
Comment (wafa200099) [inline css/list.css:1]: its better to name  the CSS file as style.css

File: list.html
Comment (wafa200099) [inline list.html:1]: it's better to name the file index.html

File: css/list.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+button,
+input,
+label {
+    cursor: pointer;
+}
+
+/******************************************************************/
+/*form search bar*/
+.search-container {
```
Comment (haneenalkilani) [inline css/list.css]: Nest styles please
```css
.search-container {
    position: relative;
    font-size: 1.4em;
    width: 100%;
    
    &  input[type="search"] {
       border: none;
       border-bottom: 1px solid var(--light-grey-shade);
       min-height: 2.4em;
       width: 100%;
       padding-inline-start: 35px;
       padding-inline-end: 10px;
       font-size: 1em;
       
       &::placeholder {
          color: var(--grey-shade);
          opacity: 0.8;
       }
    }
    
    ...
}
    
    
```

File: css/list.css [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    position: absolute;
+    top: 8px;
+    left: 8px;
+    color: var(--grey-shade);
+}
+
+
+/******************************************************************/
+/*form checkboxes*/
+.checkboxes-container {
```
Comment (haneenalkilani) [inline css/list.css]: Please utilize CSS nesting:

```
.checkboxes-container {
  width: 100%;
  margin: 20px 0px;
  border: none;

  & legend {
    display: none;
  }

  & ul {
    list-style: none;
    color: var(--grey-shade);

    & li {
    
    ...
    
    
```

File: list.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            </ul>
+        </fieldset>
+
+        <div class="buttons-container">
+            <button type="button" class="cancel" tabindex="9" onclick="cancelForm()">Cancel</button>
+            <button type="submit" class="save" tabindex="10" onclick="saveForm()">SAVE</button>
+        </div>
+    </form>
+
+    <!-- essential javascript codes to provide form functionality, note:functions are not complete yet -->
```
Comment (wafa200099) [inline list.html]: no need for any JS here its just a CSS & HTML task

**Pull request merged**